apps/monorail/views/Profiles.js

```diff
@@ -2,25 +2,12 @@
 
 'use strict';
 
-/* eslint-disable no-unused-vars */
 import React, { Component } from 'react';
-import mixin from 'common-web-ui/lib/mixin';
-import PageHelpers from 'common-web-ui/mixins/PageHelpers';
-/* eslint-enable no-unused-vars */
 
-import {} from 'material-ui';
 import ProfilesGrid from './ProfilesGrid';
 
-@mixin(PageHelpers)
 export default class Profiles extends Component {
 
-  render() {
-    return (
-      <div className="Profiles">
-        {this.renderBreadcrumbs({href: '', label: 'Dashboard'}, 'Profiles')}
-        <ProfilesGrid />
-      </div>
-    );
-  }
+  render() { return <ProfilesGrid />; }
```

Comment:
Expected an identifier and instead saw '<'.<br>Missing semicolon.<br>Expected an identifier and instead saw '>'.<br>Expected an assignment or function call and instead saw an expression.